pymongo/event_loggers.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2020-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from pymongo import monitoring
+
+"""Example event logger classes.
+.. versionadded:: 3.11
+
+These loggers can be registered using :func:`register` or 
+:class:`~pymongo.mongo_client.MongoClient`::
+    monitoring.register(CommandLogger())
+or::
+    MongoClient(event_listeners=[CommandLogger()])
+"""
+class CommandLogger(monitoring.CommandListener):
```

Comment:
Inadequate whitespace before class declaration. To avoid committing things like this, you can consider manually running flake8 or just rely on PyCharm's in-editor warnings (they are usually fairly spot-on for the style we use on this project).

Reply:
Done.